rosplan_planning_system/test/pddl/amazon/test_plan.pddl

```diff
@@ -0,0 +1,31 @@
+Number of literals: 12
```

Comment:
Is this part of the test? As the one for turtlebot below was removed.

Reply:
It is yes, is used for testing the parsing interface, without having to produce a plan.